### PR TITLE
Electrons sf

### PR DESCRIPTION
**Pull request by OlivierBondu**
Addresses partly https://github.com/cp3-llbb/Framework/issues/183:
- adds HWW and EGM SFs
- adds HWW trigger SFs and trigger list in `triggers.xml`
- adds EGM 'RECO' SFs (in practice being 'gsf tracking' SFs)

todo:
- [x] update ref trees after https://github.com/cp3-llbb/Framework/pull/201 is merged

### CODE REVIEW

**Comment by blinkseb** [comment]
Thanks a lot, looks great!

**Comment by OlivierBondu** [comment]
PR rebased and ref trees updated

File: data/ScaleFactors/Electron_EGamma_SF2D_gsfTracking.json [hunk 1]
```diff
@@ -0,0 +1,242 @@
+{
```
Comment (blinkseb) [inline data/ScaleFactors/Electron_EGamma_SF2D_gsfTracking.json:1]: This file is corrupted. I'm not sure why the Framework accept such file, but the resulting values are kind of random.

**Review by blinkseb — CHANGES_REQUESTED** [review]
The `gsfTracking` json file needs to be fixed. Otherwise thanks a lot!

File: data/ScaleFactors/Electron_EGamma_SF2D_gsfTracking.json (hunk 1 above)
Comment (OlivierBondu) [reply data/ScaleFactors/Electron_EGamma_SF2D_gsfTracking.json:1]: The original file is in the same format as others but seems to have only one bin in pt... it's probably the root cause of all this mess

Comment (OlivierBondu) [reply data/ScaleFactors/Electron_EGamma_SF2D_gsfTracking.json:1]: got it, there is a typo in the script: https://github.com/cp3-llbb/Framework/blob/69ee67e3dc6bc773164e43d692012a9dc140ab87/scripts/extractElectronScaleFactors.py#L35, fix and PR update incoming

**Comment by OlivierBondu** [comment]
PR updated with fixed json files: the bug was also removing the very first SF bin of the files... ref trees updated once more

File: data/ScaleFactors/Electron_EGamma_SF2D_gsfTracking.json (hunk 1 above)
Comment (blinkseb) [reply data/ScaleFactors/Electron_EGamma_SF2D_gsfTracking.json:1]: Thanks a lot for the fix!

**Comment by blinkseb** [comment]
Ouch, nice catch in the end!

**Review by blinkseb — APPROVED** [review]

**Pull request merged**